src/components/LiftLogContainer.tsx

```diff
@@ -18,12 +22,15 @@ const LiftLogContainer = (props: Props) => {
         <h6 className="col">Weight lifted (kg)</h6>
         <h6 className="col">Sets/Reps</h6>
       </div>
+      <AddLogEntry onAddEntry={props.onAddEntry} />
       <div className="lifts">
-        {props.entries.map((liftLogEntry, index) => (
-          <LiftRow {...liftLogEntry} key={index} />
-        ))}
+        {props.entries
+          .concat()
```

Comment:
was it not `slice` instead of `concat`?

Reply:
Good question - I googled it up. Is there any difference? Except that `slice` is easy to mix up with `splice` :)